### PR TITLE
Post review

### PR DESCRIPTION
**Pull request by kmunene**

### CODE REVIEW

File: tests/test_weconnect.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ def tearDown(self):
 class BusinessTestcase(unittest.TestCase):
     def setUp(self):
         Business.business = {}
-        self.bizna = Business("Mutura kwa Maiko", "Restaurant","Kiamaiko","Best Mutura in town")
+        self.bizna = Business("Mutura kwa Maiko", "Restaurant","Kiamaiko","Best Mutura in town","Kelvin")
```
[the remark this answers is not in the document]
Comment (kmunene) [reply tests/test_weconnect.py]: Kelvin represents the username variable, which shows the username of the person who created the business. In the views, the variable is equal to the current_user logged in when creating the business.

File: app/views.py [hunk 2]
```diff
@@ -50,4 +67,88 @@ def login():
...
+    businesses = Business.get_all_businesses() 
+    return make_response(jsonify(businesses)), 200 
+
+
+@app.route('/api/v1/businesses/<int:business_id>', methods=['PUT','GET','DELETE'])
+@jwt_required
+def one_business(business_id):
+
+    current_user = get_jwt_identity() #Current_user is username 
+    print('Current',current_user)
```
[the remark this answers is not in the document]
Comment (LWanjiru) [reply app/views.py]: What about this one?

File: tests/test_weconnect.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (LWanjiru) [reply tests/test_weconnect.py]: Oh, okay.

File: app/views.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (kmunene) [reply app/views.py]: The first line belongs there but the second doesn't, I was using it to see the value of current_user but forgot to delete it. It's now resolved.

**Pull request merged**